src/build_queue.rs

```diff
@@ -120,11 +120,8 @@ impl BuildQueue {
                 }
 
                 error!(
-                    "Failed to build package {}-{} from queue: {}\nBacktrace: {}",
-                    to_process.name,
-                    to_process.version,
-                    e,
-                    e.backtrace()
+                    "Failed to build package {}-{} from queue: {}",
+                    to_process.name, to_process.version, e,
```

Comment:
Why did you change this? I think sentry will only show the backtrace of the `error!` call, not the backtrace of where the error originally came from.